src/JWTGuard.php

```diff
@@ -181,8 +189,15 @@ public function logout($forceForever = false)
     {
         $this->requireToken()->invalidate($forceForever);
 
+        // If we have an event dispatcher instance, we can fire off the logout event
+        // so any further processing can be done. This allows the developer to be
+        // listening for anytime a user signs out of this application manually.
+        $this->events->dispatch(new Logout($this->name, $this->user));
+
         $this->user = null;
         $this->jwt->unsetToken();
+
+
```

Comment:
Can you please remove those empty lines?

Reply:
done